CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
   set(CMAKE_EXE_LINKER_FLAGS "-fuse-ld=gold -pthread")
 elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-  add_compile_definitions(WINDOWS_IGNORE_PACKING_MISMATCH="") # vs2019 complains about using #pragma pack()
+  add_compile_definitions(WINDOWS_IGNORE_PACKING_MISMATCH="") # vs complains about using #pragma pack()
```

Comment:
Unrelated to the PR, but why do we even need this one? 

We shouldn't really include win headers with custom pragma pack set, this might result in all kinds of hard to debug issues.

Reply:
Currently it won't build without it in VS. But it looks like that was just a mistake as we even have all pragma pack pops in place... They just need to be changed to pragma pack(pop, 8), as our current pragma pack(pop) is retaining previously set 1 alignment. Also I guess a bunch of those pragma's could be removed as well now.

Reply:
Changed all to pragma pack(pop, 8). Added to the to do list to reduce usage.